src/lib/ngsi/ContextElement.h

```diff
@@ -48,6 +48,10 @@ typedef struct ContextElement
   ContextAttributeVector  contextAttributeVector;  // Optional
   MetadataVector          domainMetadataVector;    // Optional
 
+  std::string             providingApplication;    // Not part of NGSI, used internally for CPr forwarding functionality
+  bool                    found;                   // Not part of NGSI, used internally for CPr forwarding functionality (update case)
+                                                   // FIXME P10 #787: not sure if we need this at the end... it only would be needed in E-<null> updates, which seems not having too much sense
```

Comment:
I cannot think any unit test leading to a CE-level `found` to `false`. So, that probably confirms that `found` is not needed at CE level. If you are not able to find any case in which this would be used, probably we should remove it and leave it only in ContextAttribute class.